Source/Microsoft.Teams.Apps.CompanyCommunicator/appsettings.json

```diff
@@ -18,7 +18,7 @@
   "MicrosoftAppPassword": "[MicrosoftAppPassword]",
   "ServiceBusConnection": "[Service Bus Connection]",
   "StorageAccountConnectionString": "[Storage Account Connection String]",
-  "DisableAuthentication": "false",
+  "DisableMustBeValidUpnCheck": "false",
```

Comment:
We're changing things so this requires a redeployment anyway. Let's then:
- change URI to Uri
- ValidUpns -> AuthorizedCreatorUpns
- DisableMustBeValidUpnCheck -> DisableCreatorUpnCheck